src/modules/dashboard/components/Extensions/Extensions.tsx

```diff
@@ -130,7 +149,9 @@ const Extensions = ({ colonyAddress }: Props) => {
                 <ExtensionCard
                   key={extension.extensionId}
                   extension={extension}
-                  installedExtension={installedExtensions[idx]}
+                  // be careful to use matching data with both
+                  // installedExtensionsData & dappInstalledExtensions
+                  installedExtension={getDappInstalledExtension(idx)}
```

Comment:
So extensions can also be installed at the contract level.

Might might want to "enforce" only showing the supported ones, even though the colony has more installed.